orttraining/orttraining/training_api/include/optimizer.h

```diff
@@ -73,6 +73,10 @@ struct Optimizer {
     return Status::OK();
   }
 
+  inline float GetLearningRate() {
```

Comment:
1) should this be const noexcept?
2) what is the purpose of the inline keyword for functions that are already inlined with the class definition?

Reply:
good, point, changing following https://en.cppreference.com/w/cpp/language/inline